jobs/ci-kubernetes-e2e-gce-large-correctness.env

```diff
@@ -0,0 +1,28 @@
+### provider-env
+KUBERNETES_PROVIDER=gce
+PROJECT=kubernetes-scale
+KUBE_GCE_ZONE=us-east1-a
+CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
+
+### cluster-env
+NUM_NODES=2000
+MASTER_SIZE=n1-standard-32
```

Comment:
Removed master size here and everywhere else as https://github.com/kubernetes/kubernetes/pull/49021 fixes it.